angular.json

```diff
@@ -33,6 +33,7 @@
             ],
             "styles": [
               "./node_modules/intl-tel-input/build/css/intlTelInput.css",
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
```

Comment:
?

Reply:
This is required for the tooltip to work correctly.